data/english.cfg

```diff
@@ -14,10 +14,9 @@ Night: −25% Damage"
 
 Day: −25% Damage
 Night: +25% Damage"
-    liminal_description= _ "Liminal units only reach their full strength during twilight.
+    liminal_description= _ "Liminal units fight best during the twilight times of day.
```

Comment:
...oh, I just noticed this. It's a string change so I guess this'll have to wait for 1.15 (_maybe_ 1.14.1 but I'm dubious on that).